README.md

```diff
@@ -146,4 +146,4 @@ This repository is used to publish a Go package providing Go bindings to the Bre
 
 Any changes to the Breez SDK, the Go bindings, and the configuration of this Go package must be made via the [breez-sdk](https://github.com/breez/breez-sdk) repo.
 
-To release a new version of this package, go to the Actions tab of this GitHub repository. Then select the *Publish Go Package* workflow and run it on the `main` branch. It will ask for a version as input. This allows you to specify the version (e.g., *0.0.1*) of the [breez-sdk](https://github.com/breez/breez-sdk) repository that should be released as a Go package.
\ No newline at end of file
+To release a new version of this package, go to the Actions tab of this GitHub repository. Then select the *Publish Go Package* workflow and run it on the `main` branch. It will ask for a version as input. This allows you to specify the version (e.g., *0.0.1*) of the [breez-sdk](https://github.com/breez/breez-sdk) repository that should be released as a Go package.
```

Comment:
I see a line change here, so I'll comment:

```suggestion
To release a new version of this package, go to the Actions tab of the https://github.com/breez/breez-sdk GitHub repository. Then select the *Publish All Packages* workflow and fill in the form with the required version. 
```

Reply:
Not sure how that line got recommitted, but happy you picked up on it!